src/script/script.cpp

```diff
@@ -138,6 +138,9 @@ const char* GetOpName(opcodetype opcode)
     case OP_NOP9                   : return "OP_NOP9";
     case OP_NOP10                  : return "OP_NOP10";
 
+    // taproot
```

Comment:
Unify Taproot capitalization in comments.

```suggestion
    // Taproot
```

Reply:
Updated.